code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -194,3 +194,11 @@
 	materials = list(/datum/material/biomass= 300)
 	build_path = /obj/item/clothing/head/rice_hat
 	category = list("initial","Organic Materials")
+
+/datum/design/paper
+	name = "Sheet of Paper"
+	id = "paper"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass= 10)
+	build_path = /obj/item/stack/sheet/paper
```

Comment:
It should be(I think) /obj/item/paper
I cannot suggest a change, so make it yourself.
Where did you got this path anyway?

Reply:
I think they just copied an earlier path and replaced leather or something with paper, but paper has a completely different object path :p